package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "usabilla-api",
-  "version": "1.0.0-beta.1",
+  "version": "1.0.0-beta.2",
```

Comment:
Version is better to be updated with another PR. First we get all the code that needs to go to another version in master and then we can create a PR to update version.